test/shared/src/main/scala/zio/test/mock/TestRandom.scala

```diff
@@ -20,7 +20,7 @@ import zio._
 import zio.random.Random
 import zio.test.mock.TestRandom.Data
 
-final case class TestRandom(ref: Ref[TestRandom.Data]) extends Random.Service[Any] {
+final case class TestRandom(private val ref: Ref[TestRandom.Data]) extends Random.Service[Any] {
```

Comment:
I would not make this private. Let's just call them, `randomState`, `clockState`, `systemState`, `consoleState`, etc.